File: PYTHON/DATASTRUCTURES/fib.py
def fib(n):
    first = 0
    second = 1
    count = 2
    array = [first, second]

    while count <= n:
        next_num = first + second
        first = second
        second = next_num
        array.append(next_num)
        count += 1
    
    return array

print(fib(10))

File: NODE/ALGORITHMS/fib.js
var fib = function (n) {
	let first = 0;
	let second = 1;
	let next;
	let count = 2;
	let result = [first, second];

	if (n < 2) {
		return "not valid n";
	}
	while (count++ < n) {
		next = first + second;
		first = second;
		second = next;
		result.push(next);
	}
	return result;
}

console.log(fib(10));